caddyhttp/proxy/upstream_test.go

```diff
@@ -448,3 +450,42 @@ func TestHealthCheckPort(t *testing.T) {
 	})
 
 }
+
+func TestHealthCheckContentString(t *testing.T) {
+	testString := "good"
+	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		fmt.Fprintf(w, "blablabla%sblablabla", testString)
+		r.Body.Close()
+	}))
+
+	_, port, err := net.SplitHostPort(server.Listener.Addr().String())
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	defer server.Close()
+
+	tests := []struct {
+		config string
+	}{
+		{"proxy / localhost:8080 {\n health_check / health_check_port " + port + " / health_check_contentstring " + testString + "\n}"},
```

Comment:
Definitely not. Does that work? 😳 

I thought we were talking about within the proxy directive only...